db/seeds/base/add_applications.rb

```diff
@@ -73,15 +73,53 @@
     # users with one withdrawn application each
     FactoryBot.create_list(
       :application,
-      4,
+      2,
       :withdrawn,
+      %i[accepted rejected].sample,
       :with_random_user,
       :with_random_work_setting,
-      :with_random_lead_provider_approval_status,
       :with_random_participant_outcome_state,
       lead_provider:,
       course: Course.all.sample,
       cohort: Cohort.all.sample,
     )
+
+    # users with one eligible for funded place application each (cohort funding_cap true)
+    FactoryBot.create_list(
+      :application,
+      2,
+      :eligible_for_funded_place,
+      :with_random_user,
+      :with_random_work_setting,
+      :with_participant_id_change,
+      lead_provider:,
+      course: Course.all.sample,
+    )
+
+    # users with one eligible for funded place application each (cohort funding_cap false)
+    FactoryBot.create_list(
+      :application,
+      2,
+      :eligible_for_funded_place,
+      :with_random_user,
+      :with_random_work_setting,
+      :with_participant_id_change,
+      lead_provider:,
+      course: Course.all.sample,
+      cohort: Cohort.where.not("start_year = ?", 2024).sample,
```

Comment:
instead of using 2024, can we use `funding_cap: false` instead here to avoid hardcoding years?